build/dependencies.props

```diff
@@ -5,7 +5,7 @@
     <AppInsightsVersion>2.1.0-beta3</AppInsightsVersion>
```

Comment:
`common.props` doesn't have the $(TargetFrameworkIdentifier) like [this](https://github.com/aspnet/CORS/pull/112/files#diff-9780bf7a72094f1ac4b023e438b548baR19). Is there some other change as well?